assets/stylesheets/dashboard.scss

```diff
@@ -173,3 +173,12 @@ h2 .fa {
     margin-bottom: 10px;
     margin-left: 5px;
 }
+
+.top-rigth {
+    top: 0px;
+    right: 0px;
+}
+
+.pt-2-and-half{
```

Comment:
Please add a space before the curly bracket.

Reply:
Also I moved this to test-details.css i think is more appropiate